pkg/appgw/health_probes.go

```diff
@@ -70,9 +70,9 @@ func (builder *appGwConfigBuilder) HealthProbesCollection(ingressList [](*v1beta
 }
 
 func (builder *appGwConfigBuilder) generateHealthProbe(backendID backendIdentifier) *network.ApplicationGatewayProbe {
-	probe := defaultProbe()
 	service := builder.k8sContext.GetService(backendID.serviceKey())
 	if service != nil {
```

Comment:
We could eventually reverse this - and say `if service == nil { return nil }` - removes an indentation level